client/src/initial/landingpagetanu/tanulanding.scss

```diff
@@ -0,0 +1,120 @@
+.page-container {
+  background: repeating-linear-gradient(35deg, #c572f8 0px 2px, #b954f8 0px 4px);
+
+  top: 0;
+  height: 100vh;
+}
+.title-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  padding-top: 8%;
+  .under-construction-text {
+    color: rgb(48, 47, 47);
+    font-style: bold;
+    text-align: center;
+    font-size: 80px;
+
+    background: repeating-linear-gradient(145deg, #ecf023 0px 8px, #f3f70d 0px 12px);
+    padding: 20px;
+    margin-bottom: 24px;
+    text-shadow: 0px 2px 4px #000000;
+  }
+  .subtitle-text {
+    color: #121212;
+    font-style: bold;
+    text-align: center;
+    font-size: 40px;
+    margin-bottom: 24px;
+  }
+}
+
+.hrline {
+  background: repeating-linear-gradient(45deg, #ecf023 0px 6px, #f3f70d 0px 12px);
+  margin-top: 5%;
+  align-items: center;
+  height: 1vh;
+  border: transparent;
+}
+.social-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  width: 100%;
+  .insta-logo {
+    padding-right: 2%;
+  }
+
+  .froshweek {
+    display: flex;
+    flex-direction: row;
+    padding-top: 5%;
+    color: #121212;
+    font-style: bold;
+    text-align: center;
+    font-size: 25px;
+    .froshweek-text {
+      padding-top: 1%;
+    }
+  }
+  .froshnomore {
+    display: flex;
+    flex-direction: row;
+    padding-top: 5%;
+    color: #121212;
+    font-style: bold;
+    text-align: center;
+    font-size: 25px;
+    margin-bottom: 24px;
+    .froshnomore-text {
+      padding-top: 1%;
+    }
+  }
+}
+
+@media screen and (max-width: 800px) {
```

Comment:
we actually have predefined types for mobile/tablet!
take a look at this file, `client\src\scssStyles\_mixins.scss`

you can import it into your .scss file, `@import '../../scssStyles/mixins';` and use this code block instead

```
@include devices(tablet) {
    // your style
  }
}
```

note that both tablet and mobile are the same sizes, so `@include devices(tablet)` is the exact same as `@include devices(mobile)`